src/name_bazaar/ui/events.cljs

```diff
@@ -192,10 +209,32 @@
  interceptors
  (fn [{:keys [db]}]
    (let [addr (get-in db [:active-page :route-params :user/address])]
-     (info ["Try address" addr (web3/address? addr)])
+     (info ["TRY ADDR RESOLUTION" addr (web3/address? addr)])
      {:db db
       :dispatch
       (if-not (web3/address? addr)
         (let [addr-patched (tldize addr)]
           [:ens.records/load [(namehash addr-patched)] {:load-resolver? true}])
-        [:public-resolver.record.addr/loaded])})))
+        [:public-resolver.addr.record/load addr])})))
+
+(reg-event-fx
+ :try-resolving-my-address
```

Comment:
This should be called `:try-resolving-my-addresses` and try to resolve all `:my-addresses`.